CHANGES.md

```diff
@@ -423,6 +423,11 @@ OpenSSL 3.0
   
    *Rich Salz*
 
+ * LHASH macros have better documentation, and OpenSSL source only
+   defines LHASH tables where the tables are actually used.
```

Comment:
IIUC this actually removes the implementation of the inline functions from the public headers in a no-deprecated build. I think there should be some discussion of that in CHANGES, and what applications that rely on this should do instead. Should inline functions implemented in the public functions be declared as deprecated as well? Otherwise there will be no indication to users of these functions that they are using something that is likely to disappear in the future.

Reply:
This follows the same wording that I used in #10669, which was merged seven months ago. The manpage in this PR explains things.  I'm happy to make a change here (and for the STACK entry as well), is it enough to point to the manpage?

I didn't think of deprecating the inline functions, and would rather not do that here as this has waited long enough.

Reply:
Actually, I think the big problem here is that application authors can't do the right thing so its not clear what the CHANGES entry should direct them to do (see my other comment).

Reply:
I still think you are wrong.  Can you sketch out an example that shows your point? Applications that want to work in both worlds can still use ```DEFINE_LHASH_OF``` in a header file. Recall that these are *inline* functions, so there's nothing in openssl libs that matters.